.github/workflows/build-dev-image.yaml

```
@@ -0,0 +1,18 @@
+name: Build Dev Image
+
+on:
+  push:
+    branches:
+      - '**'
```

Are we using this for tests only? If yes, should we instead exclude the protected branches, something like

```suggestion
on:
  push:
    branches-ignore:
    - development/**
    - q/*/**
```

or, if we use the main branch:

```suggestion
on:
  push:
    branches-ignore:
    - main
```

---

We do not have protected branches here. No Berte will be used in this repo. For now I would like them to run on Main branch as well.